src/components/App/App.tsx

```diff
@@ -59,10 +60,10 @@ class App extends React.Component<AppProps, AppState> {
     super(props)
     this.state = {
       articles: articles,
-      userSentiment: ''
+      userSentiment: null
     }
   }
-  changeUserSentiment = (newSentiment: number | string) => {
+  changeUserSentiment = (newSentiment: number | null) => {
```

Comment:
Could our default for the new sentiment be 0 so our type will always be number?

Reply:
One thing to consider with making it 0 is that zero means something in your application. 0 is a value that could mean it has a sentiment value of 0 or does it not have a sentiment? Null can be a way to say it does not have a value, and then you can determine how to handle that scenario in your app.